index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://component-site-fitui.netlify.app/components/components.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://code.iconify.design/2/2.1.2/iconify.min.js"></script>
+</head>
+
+<body>
+    <div class="grid-layout">
+        <div class="navbar">
+            <h1 class="navbar-title">Quiz</h1>
+            <div class="navbar-icons">
+                <a href="#" class="btn btn-link">dashboard</a>
+                <a href="#">
+                    <div class="avatar avatar-txt avatar-ss">user</div>
+                </a>
+            </div>
+        </div>
+        <main class="main-left">
+            <h1 class="ft-w-900">Quiz</h1>
+            <p class="ft-w-500 ft-grey">Lets play quiz!</p>
+            <a href="#quiz" class="btn btn-primary btn-ls">Explore</a>
+        </main>
+        <main class="main-right">
+            <img src="./assests/images/Questions-cuate.svg" alt="">
+        </main>
```

Comment:
give alt text to img, its a good practice

Reply:
alt added